shared/components/DemoApp/AsyncPostsRoute/Post/Post.js

```diff
@@ -78,12 +78,14 @@ function Post({ jobResult }) {
 export default withJob({
   work: (props) => {
     const { id } = props.match.params;
-    return fetch(`http://jsonplaceholder.typicode.com/posts/${id}`).then(r => r.json());
+    return fetch(`http://jsonplaceholder.typicode.com/posts/${id}`)
```

Comment:
The problem is still here. fetch function is undefined server side. You should use axios to perform the query as it works on both side. If you do, then the SSR works just fine.